pyiron_contrib/__init__.py

```diff
@@ -1,22 +1,25 @@
-from pyiron_contrib.image.job import ImageJob
-# from pyiron_contrib.protocol.generic import Protocol
-from pyiron_contrib.protocol.compound.minimize import ProtocolMinimize
-from pyiron_contrib.protocol.compound.molecular_dynamics import ProtocolMD
-from pyiron_contrib.protocol.compound.nudged_elastic_band import ProtocolNEB
-# from pyiron_contrib.protocol.compound.tild import HarmonicTILD, VacancyTILD
-# from pyiron_contrib.protocol.compound.finite_temperature_string import StringRelaxation, VirtualWork, \
-#     Milestoning, VirtualWorkParallel, VirtualWorkSerial, VirtualWorkFullStep
-from pyiron_contrib.protocol.compound.qmmm import ProtocolQMMM
-from pyiron_contrib.protocol.compound.tild import ProtocolHarmonicTILD, ProtocolHarmonicTILDParallel
+from pyiron import Project
+from pyiron.base.job.jobtype import JOB_CLASS_DICT
```

Comment:
Please be aware that `pyiron.base` changes to `pyiron_base` - most likely on Friday https://github.com/pyiron/pyiron/pull/952

Reply:
We can leave this pull request open till then, I reckon? Or I can open an issue on `pyiron_contrib` as a reminder?

Reply:
Either way is fine for me. As it is a major change I wanted to announce it in this weeks pyiron meeting before merging it.

Reply:
I will merge and close now as this is relevant for UBC. We'll make a fresh pull once the change is in place